pkg/spinup/container.go

```diff
@@ -6,15 +6,165 @@ import (
 	"strings"
 )
 
+type ContainerEvent struct {
+	CreatedAt string
+	ID        string
+	Message   string
+}
+
+type ContainerHealthCheck struct {
+	Command     []string
+	Interval    int64
+	Retries     int64
+	StartPeriod int64
+	Timeout     int64
+}
+
+type ContainerMountPoint struct {
+	ContainerPath string
+	ReadOnly      bool
+	SourceVolume  string
+}
+
+type ContainerPortMapping struct {
+	ContainerPort int64
+	HostPort      int64
+	Protocol      string
+}
+
+type Container struct {
+	ContainerArn      string
+	Cpu               string
+	ExitCode          string
+	HealthStatus      string
+	Image             string
+	LastStatus        string
+	Memory            string
+	MemoryReservation string
+	Name              string
+	NetworkBindings   []struct {
+		BindIP        string
+		ContainerPort int64
+		HostPort      int64
+		Protocol      string
+	}
+	NetworkInterfaces []struct {
+		AttachmentId       string
+		Ipv6Address        string
+		PrivateIpv4Address string
+	}
+	Reason    string
+	RuntimeId string
+	TaskArn   string
+}
+
+type ContainerTask struct {
+	Failures []string
+	Tasks    []struct {
+		AvailabilityZone string
+		Attachments      []struct {
+			Details []*NameValue
+			Id      string
+			Status  string
+			Type    string
+		}
+		CapacityProviderName  string
+		ClusterArn            string
+		Connectivity          string
+		ConnectivityAt        string
+		Containers            []*Container
+		Cpu                   string
+		CreatedAt             string
+		DesiredStatus         string
+		ExecutionStoppedAt    string
+		Group                 string
+		HealthStatus          string
+		InferenceAccelerators []struct {
+			DeviceName string
+			DeviceType string
+		}
+		LastStatus        string
+		LaunchType        string
+		Memory            string
+		Overrides         interface{}
+		PlatformVersion   string
+		PullStartedAt     string
+		PullStoppedAt     string
+		StartedAt         string
+		StartedBy         string
+		StopCode          string
+		StoppedAt         string
+		StoppedReason     string
+		StoppingAt        string
+		Tags              []*NameValue
+		TaskArn           string
+		TaskDefinitionArn string
+		Version           int64
+	}
+}
+
+type ContainerDefinition struct {
```

Comment:
Wow, big structs 😮 If any of those keys change in AWS this shouldn't break, right? Say, `Memory` changes to `MemorySize`, then the CLI is just going to return an empty value for `Memory`, I think.

Reply:
right - should jus tbe empty